packages/p2p-v2/src/components/Modals/FilterModal/FilterModal.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from 'react';
+import ReactModal from 'react-modal';
```

Comment:
can we use the the modal component from deriv-com/ui

Reply:
ah yeah, ill migrate it

Reply:
i dont think there would be a lot to change, since you have already separated the header, content and footer.. the one in deriv-com/ui uses the same format as ours.. except that the header, footer and content is separate..

Reply:
i did try, but a lot of the styles were messed up because of the footer and header, hence why I think its better to do it separately

Reply:
hmmm let me try, cause i was having issues with the styles being messed up because of the Modal Footer and header